oak/server/oak_node.cc

```diff
@@ -143,4 +147,43 @@ bool OakNode::WaitOnChannels(std::vector<std::unique_ptr<ChannelStatus>>* status
   }
 }
 
+void OakNode::Write(Handle handle, std::unique_ptr<Message> message) const {
+  {
+    absl::ReaderMutexLock lock(&labels_mu_);
+    message->labels.secrecy_tags.insert(labels_.secrecy_tags.begin(), labels_.secrecy_tags.end());
```

Comment:
I think we should have methods to convert between the C++ and the protobuf representation of labels.